src/components/modal/index.tsx

```diff
@@ -22,6 +22,8 @@ export interface ModalProps {
   style: ModalStyle
   // Vertical overflow
   verticalOverflow?: ModalOverflow
+  // Title in the modal
+  title?: ReactNode
```

Comment:
do we have different sizes/styles of titles or could we just go with a text prop? that would allow us to harmonize the title without duplicating any markup in the projects

Reply:
In all the modals in listings and DH the markup is:
`<h2 className="font-bold text-xl text-grey-dark w-12/12">` So, now we are not duplicating that markup. We pass only the translation.

Reply:
I have changed the type of the prop to a `string`